### DISCUSSION TITLE
Notifications for Unattended-upgrades

### DISCUSSION THREAD

**Discussion by bonest** [postalserver/postal#2543] [Help with using Postal]
Hi 

I am installing unattended-upgrades on my postal server, and then I thought about the email notifications that you would normally have postfix to handle. But PostalServer is a smtp server, so would you install postfix and smtp relay to localhost in order to get your postal server to send the notifications?

Are you using unattended-upgrades on your postal server, and how do you handle notification emails from unattended-upgrades?

Cheers.

**Comment by willpower232** [comment]
You could certainly do that but you may want to take a look at msmtp

https://dt.iki.fi/msmtp
https://wiki.archlinux.org/title/Msmtp

It provides the traditional email sending binaries but just forwards the messages over SMTP without having to run a whole email server on your server just to forward emails.

**Reply by bonest** [reply]
Hi @willpower232 

You are so right. It felt wrong to install postfix on a postalserver, like installing 2 different mailservers, but postfix was what was typically recommended for Unattended-upgrades, so I thought I had no choice.

I will do some testing on how to achieve this, and update the thread with my experience.

**Reply by willpower232** [reply]
I am also slowly converting from postfix so you're not alone! The trick involves setting up the aliases as you would with postfix and linking that to msmtp, I have found one cron which refuses to email a real email address but mostly painless as a concept.

**Reply by bonest** [reply]
Do you have an example describing how to link postfix aliases to msmtp?

**Reply by willpower232** [reply]
I saw this in the archlinux docs

![image](https://github.com/postalserver/postal/assets/1619102/59c93ff9-1d71-4b21-855e-88620559548f)
